Seminar1/Task2.py

```diff
@@ -0,0 +1,21 @@
+# 2.Треугольник существует только тогда, когда сумма любых двух 
+# его сторон больше третьей. Дано a, b, c - стороны предполагаемого 
+# треугольника. Требуется сравнить длину каждого отрезка-стороны с 
+# суммой двух других. Если хотя бы в одном случае отрезок окажется 
+# больше суммы двух других, то треугольника с такими сторонами не существует. 
+# Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним
+
+a = 13
+b = 11
+c = 12
+
+if a + b < c or a + c < b or b + c < a:
+    print("треугольника с такими сторонами не существует")
+elif a == b and b == c and a == c:
+    print("равносторонний")
+elif a == b or b == c or a == c:
+    print("равнобедренный")
+else:
+    print("разносторонний")
```

Comment:
Можно слегка поменять условие
```suggestion
elif a != b != c:
    print("разносторонний")
else:
    print("равнобедренный")
```